lib/contracts/flowdock-thread.ts

```diff
@@ -20,13 +20,19 @@ export const flowdockThread: ContractDefinition = {
 						id: {
 							type: 'string',
 						},
+						urls: {
+							type: 'array',
```

Comment:
@iamsolankiamit Why does a thread need multiple URLs? I was thinking it would have a single URL that corresponds to the thread's flowdock URL.

Reply:
this is to handle the multiple names of a flow. so let's say a flow is named `landr` today so all the urls will have `landr` in them, after a while it is renamed to `p/landr`. if we are generating the URL we have to store `landr` and `p/landr`, since we don't know when was the flow name changed. We also don't know the url of each thread, we have to manually construct the URL based on the flow name and thread id.